src/contexts/RoomContext.ts

```diff
@@ -40,6 +40,8 @@ const RoomContext = createContext<IState>({
     canReact: false,
     canReply: false,
     layout: Layout.Group,
+    singleSideBubbles: false,
+    adaptiveSideBubbles: false,
```

Comment:
I am probably totally overlooking where this is used

Reply:
Once a build failed if `singleSideBubbles` wasn't there. Recently I noticed that `adaptiveSideBubbles` was missing and added it just in case. But I have absolutely no clue why this is needed either.

Reply:
_Odd_